File: task-03/javascript/Sub-Task2/code2.js
const fs = require('fs');

// File paths
const inputFile = 'input.txt';
const outputFile = 'output.txt';

// Read from input.txt
fs.readFile(inputFile, 'utf8', (err, data) => 
{
    if (err) 
    {
        console.error(`Error reading ${inputFile}: ${err.message}`);
        return;
    }

    // Write to output.txt
    fs.writeFile(outputFile, data, 'utf8', (err) => 
    {
        if (err) 
        {
            console.error(`Error writing ${outputFile}: ${err.message}`);
            return;
        }

        console.log(`Content has been copied from ${inputFile} to ${outputFile}`);
    });
});

File: task-03/Go/Sub-Task2/code2.go
package main

import 
(
    "fmt"
    "io/ioutil"
    "os"
)

func main() 
{
    // Read from input.txt
    inputFile := "input.txt"
    content, err := ioutil.ReadFile(inputFile)
    if err != nil 
    {
        fmt.Printf("Error reading %s: %v\n", inputFile, err)
        return
    }

    // Write to output.txt
    outputFile := "output.txt"
    err = ioutil.WriteFile(outputFile, content, 0644)
    if err != nil 
    {
        fmt.Printf("Error writing %s: %v\n", outputFile, err)
        return
    }

    fmt.Printf("Content copied from %s to %s\n", inputFile, outputFile)
}

